contrib/nuke-graph-directory.sh

```diff
@@ -54,9 +54,11 @@ if command -v btrfs > /dev/null 2>&1; then
 	# Find btrfs subvolumes under $dir checking for inode 256
 	# Source: http://stackoverflow.com/a/32865333
 	for subvol in $(find "$dir" -type d -inum 256 | sort -r); do
-		( set -x; btrfs subvolume delete "$subvol" )
+		if [ "$dir" != "$subvol" ]; then
+			( set -x; btrfs subvolume delete "$subvol" )
+		fi
 	done
 fi
 
 # finally, DESTROY ALL THINGS
-( set -x; rm -rf "$dir" )
+( set -x; rm -rf "$dir/*" )
```

Comment:
That might have failed on me, thanks for pointing it out. What about `rm -rf "$dir"/* "$dir"/.*`?